src/test/java/com/commercetools/sync/categories/CategorySyncTest.java

```diff
@@ -45,6 +48,23 @@
     private List<String> errorCallBackMessages;
     private List<Throwable> errorCallBackExceptions;
 
+    // protected method access helper
+    private static class CategorySyncTestie extends CategorySync {
+
+        CategorySyncTestie(@Nonnull final CategorySyncOptions syncOptions,
+                                  @Nonnull final TypeService typeService,
```

Comment:
Nitpick: it would be nice of the params have the same indentation.

Reply:
Corrected